core/templates/dev/head/pages/topic-editor-page/topic-editor-page.constants.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
```

Comment:
Year :)

Reply:
FYI, don't worry about year. I think we should either keep it in sync everywhere or not worry about it.